tests/tests/test_monitor_client.py

```diff
@@ -1439,3 +1439,54 @@ def test_monitorclient_send_saved_alerts_on_network_issues(
         logger.info(
             "test_monitorclient_send_saved_alerts_on_network_issues: got OK alert email."
         )
+
+    def test_monitorclient_send_configuration_data(
+        self, monitor_commercial_setup_no_client
+    ):
+        """Tests the monitor client configuration push"""
+        wait_for_alert_interval_s = 8
+        user_name = "some.user+{}@example.com".format(str(uuid.uuid4()))
+        devid, _, auth, mender_device = self.prepare_env(
+            monitor_commercial_setup_no_client, user_name
+        )
+        logger.info("test_monitorclient_send_configuration_data: env ready.")
+
+        logger.info(
+            "test_monitorclient_send_configuration_data: push configuration scenario."
+        )
+        mender_device.run(
+            "sed -i.backup -e 's/CONFPUSH_INTERVAL=.*/CONFPUSH_INTERVAL=8/' /usr/share/mender-monitor/config/config.sh"
+        )
+        mender_device.run("systemctl restart mender-monitor")
+        prepare_service_monitoring(mender_device, "crond", use_ctl=True)
+        prepare_service_monitoring(mender_device, "dbus", use_ctl=True)
+        prepare_log_monitoring(
+            mender_device, "syslog", "/var/log/syslog", "root.*access", use_ctl=True
+        )
+        prepare_log_monitoring(
+            mender_device,
+            "clientlogs",
+            "@journalctl -u mender-client -f",
+            "[Ee]rror.*",
+            use_ctl=True,
+        )
+        time.sleep(2 * wait_for_alert_interval_s)
+        device_monitor = DeviceMonitor(auth)
+        configuration = device_monitor.get_configuration(devid)
+        assert len(configuration) == 4
+        for entity in [
+            {"name": "crond.sh", "type": "service"},
+            {"name": "dbus.sh", "type": "service"},
+            {"name": "syslog.sh", "type": "log"},
+            {"name": "clientlogs.sh", "type": "log"},
```

Comment:
Are there any example use cases where the device would publish a complete descriptor at this moment?
For example:
```json
{"name": "clientlogs.sh", "type": "log", "status": "OK?", "log": {"pattern": "any err.*or[s]?", "path": "@journalctl -u trouble", "expire_seconds": 42069}}
```